docs/rules/max-nested-callbacks.md

```diff
@@ -16,59 +16,95 @@ foo(function () {
 
 ## Rule Details
 
-This rule is aimed at increasing code clarity by discouraging deeply nesting callbacks. As such, it will warn when callbacks are nested deeper than the specified limit.
+This rule enforces a maximum depth that callbacks can be nested to increase code clarity.
 
 ## Options
 
-The default max depth for this rule is 10. You can define the depth as an option by using the second argument in your configuration. For example, this sets the rule as an error (code is 2) with a maximum depth of 3:
+This rule has a number or object option:
 
-```json
-"max-nested-callbacks": ["error", 3]
+* `"max"` (default `10`) enforces a maximum depth that callbacks can be nested
 
-// or you can use an object property
+**Deprecated:** The object property `maximum` is deprecated; please use the object property `max` instead.
 
-"max-nested-callbacks": ["error", {"max": 3}]
-```
-
-**Deprecated:** the object property `maximum` is deprecated. Please use the property `max` instead.
+### max
 
-The following patterns are considered problems:
+Examples of **incorrect** code for this rule with the default `{ "max": 10 }` option:
```

Comment:
Recommend keeping the original example with function instead of arrow function and `{ "max": 3 }` depth, even though it is not the default option.